networkx/algorithms/flow/tests/test_networksimplex.py

```diff
@@ -417,64 +417,52 @@ def test_graphs_type_exceptions():
     pytest.raises(nx.NetworkXError, nx.network_simplex, G)
 
 
-def create_graph(
-    has_large_capacity: bool = False,
-    has_large_weight: bool = False,
-    has_large_demand: bool = False,
-):
-    large_value = 1000000000
+@pytest.fixture()

```

Comment:
newer ruff will complain about the empty`()` but let's just wait for the bump anyway.